Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1170,13 +1171,14 @@ private extension PurchasesOrchestrator {
     func handlePurchasedTransaction(_ purchasedTransaction: StoreTransaction,
                                     storefront: StorefrontType?,
                                     restored: Bool) {
-        let offeringContext = self.getAndRemovePresentedOfferingIdentifier(for: purchasedTransaction)
+        let offeringContext = self.getAndRemovePresentedOfferingContext(for: purchasedTransaction)
         let paywall = self.getAndRemovePresentedPaywall()
         let unsyncedAttributes = self.unsyncedAttributes
         self.attribution.unsyncedAdServicesToken { adServicesToken in
             let transactionData: PurchasedTransactionData = .init(
                 appUserID: self.appUserID,
                 presentedOfferingID: offeringContext?.offeringIdentifier,
+                presentedPlacementID: offeringContext?.placementIdentifier,
```

Comment:
Nitpick but should we have a constructor that accepts the whole `offeringContext`?